yarn.lock

```diff
@@ -6157,28 +6157,6 @@
     "@walletconnect/types" "^1.8.0"
```

Comment:
This is fine, but should ideally be done in a separate PR.

Reply:
this wasn't intentional, I think it just happened from running yarn?